lib/vfscore/vnode.c

```diff
@@ -528,4 +528,3 @@ void vn_del_name(struct vnode *vp __unused, struct dentry *dp)
 	/* UK_ASSERT(uk_mutex_is_locked(&vp->v_lock)); */
 	uk_list_del(&dp->d_names_link);
 }
-
```

Comment:
At least the [editorconfig](https://github.com/unikraft/unikraft/blob/979f60f93275e5e833f6ad7d135a75f88d406133/.editorconfig#L9) specifies that a final newline should be inserted at the end of file.

Reply:
Hi @mschlumpp,

nice that you spotted this. However, it is fine because the file has two empty lines at the end, so VS removed one
![image](https://user-images.githubusercontent.com/8928900/154265813-08b774b6-0165-4c28-a216-def59b1bbf2c.png)

Reply:
You are right. GitHub does not show the last line...